user-org-actor/pom.xml

```diff
@@ -22,6 +22,23 @@
             <artifactId>user-org-util</artifactId>
             <version>1.0-SNAPSHOT</version>
         </dependency>
+        <dependency>
+            <groupId>io.opensaber</groupId>
+            <artifactId>registry</artifactId>
+            <version>2.0.2</version>
+        </dependency>
+        <dependency>
```

Comment:
redundant?

Reply:
fixed.

Reply:
This is compiled jar name - needs fix in user-registry repo.